src/lib/ai-service.ts

```
@@ -0,0 +1,125 @@
+
+  try {
+    switch (provider) {
+      case "openrouter":
+        return await askOpenRouterAgent({
+          messages,
+          model: selectedModel
+        });
+
+      case "gemini":
+        // For Gemini, we need to format messages differently
+        const lastMessage = messages[messages.length - 1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)